docs/components.rst

```diff
@@ -2,9 +2,128 @@
+them is deleted, `Cowbird` will not change the file permissions since other ``read`` permissions will still be found on
+`Magpie`_. This means that a synchronization will not be triggered and `Magpie`_ permissions will stay the same, meaning
+all the ``read`` permissions activated except for the one removed.
+If eventually a change is applied to the file (ex.: changing the permissions from ``r--`` to ``rw-``),
+it would trigger a synchronization, and the one `Magpie`_ permission that was removed earlier would be reenabled,
+because of the ``read`` permission found on the file.
+The same would apply if we use ``write`` permissions in this last example.
+
+Shapefiles will only be assigned ``read`` or ``write`` permissions on the file system. ``execute`` permissions are not
+needed for shapefiles.
+
+.. _geoserver_folder_workspace_permissions:
+
+Folder/Workspace permissions
+############################
+
+Workspaces will always keep their ``execute`` permissions even if they don't have any permissions enabled on `Magpie`_.
+This enables accessing the children files, in case the children resource has permissions enabled on `Magpie`_.
+Since a children resource has priority on `Magpie`_ if its permissions are enabled, it makes sense to allow the access
+to the file on the file system too. Note that if the folder only has ``execute`` permissions, the file will only be
+accessible via a direct path or url, and it will not be accessible via a file browser, or on the JupyterLab file
+browser. This should allow the user to still share his file using a path or url.
+
+.. _geoserver_operations_to_avoid:
+
+Operations to avoid
+###################
+
+Note also that some operations should be avoided, as they are undesirable and not supported for now.
+
+- Renaming a folder :
+    The folders associated with the `Geoserver` workspace are the user workspace (named by the user's name)
+    and the datastore folder (which uses a default value. Both of these values should never change and renaming them
+    manually might break the monitoring, preventing `Cowbird` from receiving future file events.
+
+- Renaming a shapefile (.shp only) :
+    This operation is actually supported, but it should be avoided if possible. It will trigger multiple events on the
+    file system (an update on the parent directory, and a delete followed by a create event on the file), which should
+    keep up to data info in `Geoserver` and `Magpie`_ by simply generating new resources. A risk with this is that the
+    delete event will delete the other shapefile files, and the user could lose some data. It is better to have a copy
+    of the shapefile before applying this operation. Note that renaming one of the other extension (not the .shp file)
+    will not trigger any event since only the main file triggers events.
+
+- Deleting a folder :
+    This operation will only display a warning. It should never be done manually, since it will create inconsistencies
```

Comment:
Display a warning (in logs). Clarify, since the reader could think a popup would appear in JupyterHub.